code/datums/martial/krav_maga.dm

```diff
@@ -86,7 +86,7 @@
 	D.visible_message("<span class='warning'>[A] pounds [D] on the chest!</span>", \
 				  	"<span class='userdanger'>[A] slams your chest! You can't breathe!</span>")
 	playsound(get_turf(A), 'sound/effects/hit_punch.ogg', 50, 1, -1)
-	D.losebreath += 5
+	D.losebreath = Clamp(D.losebreath + 5, 0, 10)
```

Comment:
I'm pretty sure this lowers losebreath to 10 if it is higher than 10 before the punch.
I think the easiest way to do this would be to put this in a conditional.